libraries/chain/webassembly/privileged.cpp

```diff
@@ -171,10 +179,25 @@ namespace eosio { namespace chain { namespace webassembly {
 
       std::vector<abi_finalizer_authority>& finalizers = abi_finpol.finalizers;
 
-      EOS_ASSERT( finalizers.size() <= config::max_finalizers, wasm_execution_error, "Finalizer set exceeds the maximum finalizer count for this chain" );
-      EOS_ASSERT( finalizers.size() > 0, wasm_execution_error, "Finalizer set cannot be empty" );
+      EOS_ASSERT( finalizers.size() <= config::max_finalizers, wasm_execution_error, "Finalizer policy exceeds the maximum finalizer count for this chain" );
+      EOS_ASSERT( finalizers.size() > 0, wasm_execution_error, "Finalizers cannot be empty" );
+
+      struct g1_hash {
+         size_t operator()(const bls12_381::g1& g1) const {
+            size_t hash = 0;
+            hash ^= fp_hash(g1.x) + 0x9e3779b9 + (hash << 6) + (hash >> 2);
+            hash ^= fp_hash(g1.y) + 0x9e3779b9 + (hash << 6) + (hash >> 2);
+            hash ^= fp_hash(g1.z) + 0x9e3779b9 + (hash << 6) + (hash >> 2);
+            return hash;
+         }
+      };
```

Comment:
These lists are going to be small, we mostly are concerned about just providing the correct `equal`. This is likely an over complex hash function for what is needed. Likely we could just do `return g1.y.d[0]`, although I'm not sure what would be best to pick. @arhag

Reply:
Thanks. I will go with simple method.